src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -26,6 +27,57 @@ public enum BusChain {
 		this.chainName = chainName;
 		this.logPath = LOG_PATH_PREFIX + this.chainName + "/";
 		this.busStatus = CANBus.getStatus(this.chainName);
+
+		createAlerts();
+	}
+
+	private void createAlerts() {
+		//@formatter:off
+		AlertManager.addAlert(
+				new PeriodicAlert(
+						Alert.AlertType.WARNING,
+						logPath + "StatusErrorAt",
+						() -> !busStatus.Status.isOK()
+				)
+		);
+		//@formatter:on
+		AlertManager.addAlert(
+				new PeriodicAlert(
+						Alert.AlertType.WARNING,
+						logPath + "FloodedAt",
+						() -> busStatus.BusUtilization > PERMITTED_CAN_UTILIZATION_DECIMAL_VALUE
+				)
+		);
+		//@formatter:off
+		AlertManager.addAlert(
+				new PeriodicAlert(
+						Alert.AlertType.WARNING,
+						logPath + "ReceiveErrorAt",
+						() -> busStatus.REC > PERMITTED_RECEIVE_ERRORS
+				)
+				//@formatter:on
```

Comment:
why stop formatter in the middle of `addAlert()`?

Reply:
puts it all in one line